src/fides/api/tasks/__init__.py

```diff
@@ -5,11 +5,19 @@
 from sqlalchemy.orm import Session
 
 from fides.api.db.session import get_db_engine, get_db_session
-from fides.api.util.logger import create_handler_dicts
 from fides.api.util.logger import setup as setup_logging
 from fides.config import CONFIG, FidesConfig
 
 MESSAGING_QUEUE_NAME = "fidesops.messaging"
+PRIVACY_PREFERENCES_QUEUE_NAME = "fides.privacy_preferences"
```

Comment:
maybe a little comment here that this is populated/used in plus would help?

it's a bit odd to me that we define this here in OSS but have basically everything else defined in plus. i think i understand why - and i think that it's probably not worth trying to re-do a lot of this functionality in plus - but i do feel like a comment here would help to explain why there's not more references to this queue name within the OSS codebase itself

Reply:
good idea 👍